CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.4-wip
```

Comment:
Do you need this published soon? It likely won't be published for some time if we don't do it now